doc/nrf/releases/release-notes-latest.rst

```diff
@@ -222,24 +227,29 @@ The following list summarizes the most important changes inherited from upstream
 * Added support for invoking shell commands (shell management) from the mcumgr command line.
 
 
+Multiprotocol Service Layer (MPSL)
+==================================
+
+* The nRF53 has been modified to use SWI by default, instead of EGU.
```

Comment:
```suggestion
* Modified the nRF5340 to use SWI by default, instead of EGU.
```
What exactly was updated here? MPSL support for nRF5340?
I doubt we updated the chip. ;-)